org.swi_prolog.swipl.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.swi_prolog.swipl</id>
+  
+  <name>SWI-Prolog</name>
+  <summary>ISO/Edinburgh-style Prolog interpreter</summary>
+  <url type="homepage">https://www.jabref.org</url>
```

Comment:
why is this not linking to swi-prolog.org ?

Reply:
This is a typo from a previous file I copied.
I'll fix it now.